telethon/extensions/binaryreader.py

```diff
@@ -123,7 +123,7 @@ def tgread_date(self):
         if value == 0:
             return None
         else:
-            return datetime.fromtimestamp(value, tz=timezone.utc)
+            return datetime.fromtimestamp(0, tz=timezone.utc) + timedelta(seconds=value)
```

Comment:
What if we just use `_EPOCH = _EPOCH_NATIVE.replace(tzinfo=timezone.utc)` here too instead of `datetime.fromtimestamp(0, tz=timezone.utc)`?